### DISCUSSION TITLE
Public werden?

### DISCUSSION THREAD

**Discussion by real-Shurakra** [real-Shurakra/Berichtshefterstellung#7] [Ideas]
Ich denke mal nicht, frage aber dennoch.
Hat jemand was dagegen, wenn ich das repo public mache?
@BenjaminWilliamAtkins @Nvasiljevic

**Comment by BenjaminWilliamAtkins** [comment]
nope nix dagegen. Von meinem Samsung Galaxy Smartphone gesendet.
-------- Ursprüngliche Nachricht --------Von: Matthias Koperski ***@***.***> Datum: 16.09.21  11:40  (GMT+01:00) An: real-Shurakra/Berichtshefterstellung ***@***.***> Cc: BenjaminWilliamAtkins ***@***.***>, Mention ***@***.***> Betreff: [real-Shurakra/Berichtshefterstellung] Public werden? (Discussion #7) 
Ich denke mal nicht, frage aber dennoch.
Hat jemand was dagegen, wenn ich das repo public mache?
@BenjaminWilliamAtkins @Nvasiljevic

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/real-Shurakra/Berichtshefterstellung/discussions/7",
"url": "https://github.com/real-Shurakra/Berichtshefterstellung/discussions/7",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by real-Shurakra** [reply]
Du solltest vtl. ausschalten, dass du die ursprüngliche Nachricht mit sendest (≧∇≦)/
